core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -97,8 +97,7 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
                  ng-click="setActiveSubSection(LEARNER_DASHBOARD_SUBSECTIONS.EXPLORATIONS)"
                  translate="I18N_DASHBOARD_EXPLORATIONS">
             </div>
-            <div id="completedSection"
-                 class="oppia-learner-dashboard-submenu"
+            <div class="oppia-learner-dashboard-submenu protractor-test-collection-section"
```

Comment:
Ideally the protractor class would be unique. How about protractor-test-incomplete-collection-session? And ditto below.